### ISSUE TITLE
[Bug]: Sveltekit with adapter-cloudflare is using the twind browser module in worker script

### ISSUE THREAD

**Issue by 0x221A** [tw-in-js/twind#418]
### Reproduction

1. Use Sveltekit with the adapter-cloudflare package
2. Import the twind module in server code
3. Call the twind `install` function
4. Deploy to a production environment

### Describe the bug

Sveltekit is using the twind browser module in worker script, which is causing an error where `document` is undefined. This error is preventing the twind `install` function from running properly.

```
ReferenceError: document is not defined
  at getStyleElement (/Users/.../node_modules/@twind/src/sheets.ts:6:15)
  at cssom (/Users/.../node_modules/@twind/src/sheets.ts:29:11)
  at getSheet (/Users/.../node_modules/@twind/src/sheets.ts:181:86)
  at sheet (/Users/.../node_modules/@twind/src/install.ts:33:11)
  at setup (/Users/.../node_modules/@twind/src/runtime.ts:130:67)
  at eV (/Users/.../node_modules/@twind/src/install.ts:27:10)
  at .svelte-kit/output/server/entries/pages/_layout.ts.js (/Users/.../.svelte-kit/output/server/entries/pages/_layout.ts.js:19:1)
  at __init (/private/var/folders/9t/gvy_vyd1721gjwcmrmlp03t80000gn/T/tmp-68294-pXtmS4gX2vT3/srrsik229vh.js:29:56)
  at .svelte-kit/output/server/nodes/0.js (/Users/.../.svelte-kit/output/server/nodes/0.js:1:1)
  at __init (/private/var/folders/9t/gvy_vyd1721gjwcmrmlp03t80000gn/T/tmp-68294-pXtmS4gX2vT3/srrsik229vh.js:29:56)
```

### Severity

serious, but I can work around it

### System Info

```sh
## Binaries:
 - Node: 16.18.1 - ~/Library/Caches/fnm_multishells/90960_1670456193433/bin/node
 - Yarn: 3.3.0 - /usr/local/bin/yarn
 - npm: 8.19.2 - ~/Library/Caches/fnm_multishells/90960_1670456193433/bin/npm
## Browsers:
 - Chrome: 108.0.5359.98
 - Safari: 16.2
## npmPackages:
 - @sveltejs/adapter-cloudflare: =1.0.0-next.43 => 1.0.0-next.43 
 - @sveltejs/kit: =1.0.0-next.587 => 1.0.0-next.587 
 - @twind/core: ^1.0.3 => 1.0.3 
 - @twind/preset-autoprefix: ^1.0.3 => 1.0.3 
 - @twind/preset-tailwind: ^1.0.3 => 1.0.3 
 - svelte: ^3.55.0 => 3.55.0 
 - vite: ^4.0.1 => 4.0.1
```

### Additional Information

After the last version of @sveltejs/adapter-cloudflare and @sveltejs/kit was released, they changed to use `esm-env` to detect the browser environment. However, the esbuild target platform was set to `browser`, which caused the use of the twind browser module in server code. This led to the error where `document` is undefined and the failure of the twind `install` function.

**Comment by sastan** [comment]
Thank you for the heads-up. I will look into it asap.

**Comment by sastan** [comment]
I implemented a fix that will be available in the next release (#417). Until that is released you can use the pre-release version   (for example [`@twind/with-sveltekit@next`](https://www.npmjs.com/package/@twind/with-sveltekit/v/1.0.4-next-20221214233525), [`@twind/core@next`](https://www.npmjs.com/package/@twind/core/v/1.1.0-next-20221214233525) or [`@twind/preset-tailwind@next`](https://www.npmjs.com/package/@twind/preset-tailwind/v/1.1.0-next-20221214233525)).

If you have any issues or questions please re-open this issue.
